src/containers/Petitions.js

```diff
@@ -12,7 +12,7 @@ const PetitionsContainer = React.createClass({
   render () {
     return (
       <div>
-        <Helmet title='Petitions' />
+        <Helmet title='Petitionen' />
```

Comment:
Same here. Plus this only does the client-side page title. The server-side page is defined in https://github.com/iris-dni/iris-frontend/blob/develop/src/server/getBrowserTitle.js :)
e.g.
```
    case 'PetitionsContainer':
      return TITLE_TEMPLATE.replace('%s', settings.petitionsTitle);
```